aws/resource_aws_ses_event_destination_test.go

```diff
@@ -26,6 +26,8 @@ func TestAccAWSSESEventDestination_basic(t *testing.T) {
 						"aws_ses_event_destination.kinesis", "name", "event-destination-kinesis"),
 					resource.TestCheckResourceAttr(
 						"aws_ses_event_destination.cloudwatch", "name", "event-destination-cloudwatch"),
+					resource.TestCheckResourceAttr(
+						"aws_ses_event_destination.sns", "name", "event-destination-sns"),
```

Comment:
Nitpick - but do you mind adding check for `sns_destination.0.topic_arn` as that is the field being added and that's the one that should also be tested?